### PR TITLE
Editorial: rewrite terminology section

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: static/specification.html [hunk 1]
```diff
@@ -98,35 +98,29 @@ <h2>
...
         <dt>
           <dfn>Web Monetization receiver</dfn>:
         </dt>
         <dd>
-          the party receiving payments on behalf of the website.
+          The party receiving payments on behalf of the website.
         </dd>
         <dt>
-          <dfn>User agent</dfn>:
+          <dfn>Payment end-point</dfn>:
```
Comment (AlexLakatos) [inline static/specification.html:119]: This is what we're calling both the "pointer" and the "href"? Should we keep this as the href, and add a separate entry for the payment pointer?

Comment (marcoscaceres) [reply static/specification.html:119]: Agree... although, I think this one is ok, as the spec explicitly states that the `href` must be a "payment end-point" (which is a SPSP end-point).

**Review by AlexLakatos — APPROVED** [review]
LGTM 🦖

File: static/specification.html (hunk 1 above)
Comment (AlexLakatos) [reply static/specification.html:119]: Great!

**Pull request merged**